08LessonEight/in-class-project/css/homework-style.css

```diff
@@ -0,0 +1,138 @@
+* {
+  font-family:sans-serif;
+  color: #636363;
+  font-weight: 100;
+}
+
+header {
+  width: 40vw;
+  margin:0 auto;
+  margin-bottom: 3%;
+  font-size: 12px;
+  text-align: center;
+}
```

Comment:
careful when mixing vw and % width as they CAN be the same but vw isn't relational like % is.